### PR TITLE
CI Test branch 2.0!

### PR DESCRIPTION
**Pull request by Asherslab**
Self explanatory

### CODE REVIEW

**Comment by BuildTeamLinter** [comment]
SonarQube analysis reported 47 issues
* ![MAJOR][MAJOR] 47 major

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [blocks#](https://github.com/ldtteam/minecolonies/blob/82578db58fa1dbf4feb49c856613e50c42f0538b/src/main/java/com/minecolonies/coremod/blocks#): Add a 'package-info.java' file to document the 'blocks' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MAJOR][MAJOR] [gui#](https://github.com/ldtteam/minecolonies/blob/82578db58fa1dbf4feb49c856613e50c42f0538b/src/main/java/com/minecolonies/coremod/client/gui#): Add a 'package-info.java' file to document the 'gui' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MAJOR][MAJOR] [render#](https://github.com/ldtteam/minecolonies/blob/82578db58fa1dbf4feb49c856613e50c42f0538b/src/main/java/com/minecolonies/coremod/client/render#): Add a 'package-info.java' file to document the 'render' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MAJOR][MAJOR] [permissions#](https://github.com/ldtteam/minecolonies/blob/82578db58fa1dbf4feb49c856613e50c42f0538b/src/main/java/com/minecolonies/coremod/colony/permissions#): Add a 'package-info.java' file to document the 'permissions' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MAJOR][MAJOR] [requestable#](https://github.com/ldtteam/minecolonies/blob/82578db58fa1dbf4feb49c856613e50c42f0538b/src/main/java/com/minecolonies/coremod/colony/requestable#): Add a 'package-info.java' file to document the 'requestable' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MAJOR][MAJOR] [data#](https://github.com/ldtteam/minecolonies/blob/82578db58fa1dbf4feb49c856613e50c42f0538b/src/main/java/com/minecolonies/coremod/colony/requestsystem/data#): Add a 'package-info.java' file to document the 'data' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MAJOR][MAJOR] [init#](https://github.com/ldtteam/minecolonies/blob/82578db58fa1dbf4feb49c856613e50c42f0538b/src/main/java/com/minecolonies/coremod/colony/requestsystem/init#): Add a 'package-info.java' file to document the 'init' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MAJOR][MAJOR] [locations#](https://github.com/ldtteam/minecolonies/blob/82578db58fa1dbf4feb49c856613e50c42f0538b/src/main/java/com/minecolonies/coremod/colony/requestsystem/locations#): Add a 'package-info.java' file to document the 'locations' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MAJOR][MAJOR] [management#](https://github.com/ldtteam/minecolonies/blob/82578db58fa1dbf4feb49c856613e50c42f0538b/src/main/java/com/minecolonies/coremod/colony/requestsystem/management#): Add a 'package-info.java' file to document the 'management' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MAJOR][MAJOR] [handlers#](https://github.com/ldtteam/minecolonies/blob/82578db58fa1dbf4feb49c856613e50c42f0538b/src/main/java/com/minecolonies/coremod/colony/requestsystem/management/handlers#): Add a 'package-info.java' file to document the 'handlers' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

File: src/main/resources/mcmod.info [hunk 1]
```diff
@@ -2,14 +2,14 @@
 {
   "modid": "minecolonies",
```
Comment (marchermans) [inline src/main/resources/mcmod.info:3]: ### Infos:
- Typo: In word 'modid'
- Typo: In word 'minecolonies'

File: src/main/resources/mcmod.info [hunk 2]
```diff
@@ -2,14 +2,14 @@
...
+  "description": " MineColoneyes is a Coloony/Town Simulater that AIDS many structures and NPC workers and guaaaards that you control and put to work for you! Based around a townhall and village mechanic, there are numerous workers that can be added to your colony, building a busy and vibrant village/town/city within Minecraft. There is no coded limit to the number of NPC's at your disposal, so you can create a small efficient Village dedicated to a particular type of worker, or a sprawling metropolis with many workers. \n It's entirely up to you! BlaBla another spellchecking erroooorrr by Minecolonies and even more qhwefiuhiufhwieufgrhiwefhgiehfiuehfh,iequrdhjfoiqehfriuheifh nedfehfierhfieurhf ewirfhieuhfiherifheif eiuhrfiuehrfiuehfieuhf.",
```
Comment (marchermans) [inline src/main/resources/mcmod.info:5]: ### Infos:
- Typo: In word 'NPC's'
- Typo: In word 'nedfehfierhfieurhf'
- Typo: In word 'qhwefiuhiufhwieufgrhiwefhgiehfiuehfh'
- Typo: In word 'erroooorrr'
- Typo: In word 'Minecraft'
- Typo: In word 'Minecolonies'
- Typo: In word 'guaaaards'
- Typo: In word 'townhall'
- Typo: In word 'ewirfhieuhfiherifheif'
- Typo: In word 'Coloneyes'
- Typo: In word 'Coloony'
- Typo: In word 'Simulater'
- Typo: In word 'eiuhrfiuehrfiuehfieuhf'
- Typo: In word 'iequrdhjfoiqehfriuheifh'

File: src/main/resources/mcmod.info [hunk 3]
```diff
@@ -2,14 +2,14 @@
...
   "version": "${version}",
   "mcversion": "${mcversion}",
```
Comment (marchermans) [inline src/main/resources/mcmod.info:7]: ### Infos:
- Typo: In word 'mcversion'

File: src/main/resources/mcmod.info [hunk 4]
```diff
@@ -2,14 +2,14 @@
...
-  "url": "http://minecolonies.com",
+  "url": "http://minecoolonies.com",
   "updateUrl": "",
-  "authorList": ["MineColonies Mod Team"],
+  "authorList": ["MineCrapponies Mod Team"],
```
Comment (marchermans) [inline src/main/resources/mcmod.info:10]: ### Infos:
- Typo: In word 'Crapponies'

File: src/main/resources/mcmod.info [hunk 5]
```diff
@@ -2,14 +2,14 @@
...
   "credits": "Lunatrius",
```
Comment (marchermans) [inline src/main/resources/mcmod.info:11]: ### Infos:
- Typo: In word 'Lunatrius'

**Review by marchermans — APPROVED** [review]
Analysis completed. 
   Found a total of: 29854 (6) inspections marked in the branch. 
   With a total of 8 (0) errors.

File: src/main/resources/mcmod.info [hunk 6]
```diff
@@ -2,14 +2,14 @@
...
-  "logoFile": "assets/minecolonies/Logo.jpg",
+  "logoFile": "asssssss/minecolonies/Logo.jpg",
```
Comment (marchermans) [inline src/main/resources/mcmod.info:12]: ### Infos:
- Typo: In word 'minecolonies'
- Typo: In word 'asssssss'

**Comment by LDTTeam-Buildsystem** [comment]
Test 123

**Pull request closed without merging**